dir.props

```diff
@@ -2,7 +2,7 @@
   <Import Condition="Exists('..\dir.props')" Project="..\dir.props" />
   <!-- Build Tools Version -->
   <PropertyGroup>
-    <BuildToolsVersion>1.0.25-prerelease-00023</BuildToolsVersion>
+    <BuildToolsVersion>1.0.25-prerelease-00025</BuildToolsVersion>
```

Comment:
the buildtools in the repo changed to 00026 (the latest), please undo this change while merging.